docs/preview/features/powershell/azure-integration-account.md

```diff
@@ -85,5 +86,74 @@ PS> Set-AzIntegrationAccountSchemas -ResourceGroupName 'my-resource-group' -Name
 ```
 
 
+## Uploading maps into an Azure Integration Account
+
+Upload/update a single, or multiple maps into an Azure Integration Account.
+
+| Parameter              | Mandatory | Description                                                                                                                            |
+| ---------------------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `ResourceGroupName`    | yes       | The name of the Azure resource group where the Azure Integration Account is located.                                                   |
+| `Name`                 | yes       | The name of the Azure Integration Account into which the maps are to be uploaded/updated.                                              |
+| `MapFilePath`          | no        | The full path of a map that should be uploaded/updated. (Mandatory if MapsFolder has not been specified).                              |
+| `MapsFolder`           | no        | The path to a directory containing all maps that should be uploaded/updated. (Mandatory if MapFilePath has not been specified).        |
+| `MapType`              | no        | The type of map to be created, default to 'Xslt'. See possible values [here](https://docs.microsoft.com/en-us/powershell/module/az.logicapp/get-azintegrationaccountmap?view=azps-6.2.1#parameters).  |
```

Comment:
Maybe these should have something else in the `Mandatory` column? Like `yes (when ... is not specified)` or `sometimes` or...

Reply:
Good point!
I'll change this to "conditional" and set the actual condition in _italic_ in the description-column.